### PR TITLE
Added file watcher in dev dependencies

### LINKED ISSUE #1829: File watcher for restarting server
Whenever a developers tries to make changes to the code, the server needs to be restarted manually. This creates a lot of headache. Why not automate the task and add a file watcher to restart the server on changes.

I am sending PR for the same :sweat_smile:

### PR DESCRIPTION
**Pull request by yashLadha**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Added a file watcher for the changes and restart the server.

Fixes #1829

### CODE REVIEW

**Comment by yashLadha** [comment]
Ping @mariobehling @geekyd

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ addons:
 install:
   - npm install -g istanbul mocha@3
   - npm install
+  - npm insatll --save-dev
```
Comment (geekyd) [inline .travis.yml]: There is a typing mistake here. It should be `install` instead of `insatll`

Comment (yashLadha) [reply .travis.yml]: My bad @geekyd :sweat_smile:

**Comment by yashLadha** [comment]
Done changes @geekyd . Please review it again.

**Pull request merged**